deps/jerry/tools/snap.js

```diff
@@ -0,0 +1,122 @@
+var fs = require('fs');
+
+const NODE_FILED_OFFSET_TYPE = 0;
+const NODE_FIELD_OFFSET_NAME = 1;
+const NODE_FIELD_OFFSET_ID = 2;
+const NODE_FIELD_OFFSET_SELF_SIZE = 3;
+
+const EDGE_FIELD_OFFSET_TYPE = 0;
+const EDGE_FIELD_OFFSET_NAME = 1;
+const EDGE_FIELD_OFFSET_TO_NODE = 2;
+
+const DUMMY_STRING = "<dummy>";
```

Comment:
Use the single quote for strings always in JavaScript.

Reply:
3x!
The eslint has already give me many warning. I'll fix them all.